File: goshow.py
import wget
url = "http://cdn.golangshow.com/episodes/"
print "Enter number of episodes to download"
from_ep = int(input("from episode: "))
to_ep = int(input("to episode: "))
for ep in range(from_ep, to_ep + 1):
    if ep < 10:
        episode = "00" + str(ep) + ".mp3"
    elif ep < 100:
        episode = "0" + str(ep) + ".mp3"
    print("episode #" + str(ep))
    filename = wget.download(url + episode)
    filename

File: goshow.rb
puts "Enter number of season"
url = "http://cdn.golangshow.com/episodes/"
puts "Enter number of episodes to download"
puts "from episode: "
from_ep = gets.to_i
puts "to episode: "
download = ''
to_ep = gets.to_i
(from_ep..to_ep).to_a.each do |ep|
  episode = "#{ep.to_s.rjust(3, '0')}.mp3"
  download += "wget #{url}#{episode};"
end
exec(download)
